turn/src/allocation/allocation_manager.rs

```diff
@@ -41,6 +44,36 @@ impl Manager {
         Ok(())
     }
 
+    // Returns the information about the all [`Allocation`]s associated with
+    // the specified [`FiveTuple`]s.
+    pub async fn get_allocations_info(
+        &self,
+        five_tuples: Option<Vec<FiveTuple>>,
+    ) -> HashMap<FiveTuple, AllocationInfo> {
+        let mut infos = HashMap::new();
+
+        let guarded = self.allocations.lock().await;
+
+        guarded.iter().for_each(|(five_tuple, alloc)| {
+            if five_tuples.is_none() || five_tuples.as_ref().unwrap().contains(five_tuple) {
+                drop(infos.insert(
```

Comment:
Why this explicit call to `drop`? If the return value isn't bound to anything this already happens

Reply:
You are right, it's redundant.